src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,147 @@
+package frc.trigon.robot.subsystems.climber;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.ElevatorSimulation;
+import org.trigon.utilities.mechanisms.ElevatorMechanism2d;
+
+public class ClimberConstants {
+    private static final int
+            RIGHT_MOTOR_ID = 1,
+            LEFT_MOTOR_ID = 12;
+    private static final String
+            RIGHT_MOTOR_NAME = "RightClimberMotor",
+            LEFT_MOTOR_NAME = "LeftClimberMotor";
+    static final TalonFXMotor
+            RIGHT_MOTOR = new TalonFXMotor(
+            RIGHT_MOTOR_ID,
+            RIGHT_MOTOR_NAME,
+            RobotConstants.CANIVORE_NAME
+    ),
+            LEFT_MOTOR = new TalonFXMotor(
+                    LEFT_MOTOR_ID,
+                    LEFT_MOTOR_NAME,
+                    RobotConstants.CANIVORE_NAME
+            );
+
+    private static final InvertedValue
+            RIGHT_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            LEFT_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    static final boolean ENABLE_FOC = true;
+    static final double GEAR_RATIO = 1; //TODO: ask mechanics for number
```

Comment:
This is after PID and FeedForward when you configure the motor so it should be after them when you create it as well